test/images/agnhost/webhook/pods.go

```diff
@@ -30,6 +30,9 @@ const (
 	podsInitContainerPatch string = `[
 		 {"op":"add","path":"/spec/initContainers","value":[{"image":"webhook-added-image","name":"webhook-added-init-container","resources":{}}]}
 	]`
+	podsSidecarPatch string = `[
+		{"op":"add", "path":"/spec/containers/-","value":{"image":"%v","name":"webhook-added-sidecar","resources":{}}}
```

Comment:
what does the pause container do when it gets a term signal? will adding it here make the pods with this injected sidecar wait for the default grace period when deleted, or does the pause container exit quickly?

Reply:
AFAIK it exits quickly.